packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -216,13 +186,25 @@ describe('Validation at REST level', () => {
               path: '/price',
               code: 'maximum',
               message: 'should be <= 100',
-              info: {comparison: '<=', limit: 100, exclusive: false},
+              info: {comparison: '<=', limit: 100},
             },
             {
               path: '/price',
-              code: 'range',
-              message: 'should pass "range" keyword validation',
-              info: {keyword: 'range'},
+              code: 'errorMessage',
+              message: 'price should be in range 0 to 100',
+              info: {
+                errors: [
+                  {
+                    keyword: 'range',
+                    dataPath: '/price',
+                    schemaPath:
+                      '#/components/schemas/Product/properties/price/range',
+                    params: {},
+                    message: 'should pass "range" keyword validation',
+                    emUsed: true,
+                  },
+                ],
+              },
```

Comment:
Hmm, from the perspective of a consumer of a LB-powered REST APIs, I find the previous error easier to understand.  Is there a way how to configure AJV to avoid wrapping `range` errors in `code: errorMessage` objects?